src/rules/selector-attribute-operator-space-after/README.md

```diff
@@ -0,0 +1,133 @@
+# selector-attribute-operator-space-after
+
+Require a single space or disallow whitespace after operator within attribute selectors.
+
+```css
+.foo[target= _blank]
+/**         ↑    
+ * The space after operator */
+```
+
+## Options
+
+`string`: `"always"|"never"`
+
+### `"always"`
+
+There *must always* be a single space inside the brackets.
```

Comment:
Should be: `There *must always* be a single space on both sides of the operator.` I think.